### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by skynet** [djchen/pocket-api-php#4]
Is this still maintained? Don't see much documentation on how to use it.

**Comment by djchen** [comment]
Yes. You're right there is no documentation. The best bet is to look at the sample.php.
If you would like to contribute some documentation, that would be greatly appreciated.
Thanks!
